packages/components/src/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Wot-Components</title>
+  <script type="module" src="/build/ui-wot-components.esm.js"></script>
+  <script nomodule src="/build/ui-wot-components.js"></script>
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script>
+    tailwind.config = {
+      darkMode: 'class',
+      theme: {
+        extend: {
+          colors: {
+            primary: '#067362',
+            secondary: '#B84A91',
+          }
+        }
+      }
+    }
+  </script>
+</head>
+<body class="bg-gray-50 dark:bg-gray-900 min-h-screen font-sans transition-colors duration-300">
+  <!-- Navigation -->
+  <nav class="bg-white dark:bg-gray-800 border-b border-gray-200 dark:border-gray-700 px-6 py-4 transition-colors duration-300">
+    <div class="flex justify-between items-center max-w-6xl mx-auto">
+      <h1 class="text-xl font-semibold text-gray-900 dark:text-white">Wot-Components</h1>
+      <div class="flex items-center gap-3">
+        <span class="text-sm text-gray-600 dark:text-gray-300">Light</span>
+        <ui-toggle id="theme-toggle" variant="apple"></ui-toggle>
```

Comment:
I prefer the simple version as a theme toggler, is also more aligned with the Thingweb style. 
```suggestion
        <ui-toggle id="theme-toggle" ></ui-toggle>
```